blocklylib-core/src/main/java/com/google/blockly/model/Block.java

```diff
@@ -366,6 +368,13 @@ public void setComment(String comment) {
     }
 
     /**
+     * @return The input for the mutator button if one exists.
+     */
+    public Field getMutatorField() {
```

Comment:
This is only called by the BlockViewFactory, making me thing it shouldn't be part of the model.